assets/sass/admin/_event_hosting.scss

```diff
@@ -0,0 +1,376 @@
+.event-hosting-specialist,
```

Comment:
We should remove this and other project specific stuff.

Reply:
done here https://github.com/netgen/media-site/commit/c4c897f58797d6772823557b30c8fcde98037e10